app/bundles/EmailBundle/EventListener/ReportSubscriber.php

```diff
@@ -159,11 +224,20 @@ public function onReportGenerate(ReportGeneratorEvent $event)
     {
         $context = $event->getContext();
         $qb      = $event->getQueryBuilder();
+        // channel_url_trackables subquery
+        $qbcut   = $this->db->createQueryBuilder();
+        $qbcut->select('SUM(cut2.hits) AS hits','SUM(cut2.unique_hits) AS unique_hits', 'cut2.channel_id')
+            ->from(MAUTIC_TABLE_PREFIX.'channel_url_trackables', 'cut2')
+            ->where('cut2.channel = \'email\'')
+            ->groupBy('cut2.channel_id');
 
         switch ($context) {
             case 'emails':
                 $qb->from(MAUTIC_TABLE_PREFIX.'emails', 'e')
-                    ->leftJoin('e', MAUTIC_TABLE_PREFIX.'emails', 'vp', 'vp.id = e.variant_parent_id');
+                    ->leftJoin('e', MAUTIC_TABLE_PREFIX.'emails', 'vp', 'vp.id = e.variant_parent_id')
+                    ->leftJoin('e', sprintf('(%s)', $qbcut->getSQL()), 'cut', 'e.id = cut.channel_id')
+                    ->leftJoin('e', MAUTIC_TABLE_PREFIX.'lead_donotcontact', 'dnc', 'e.id = dnc.channel_id and dnc.channel=\'email\' and dnc.reason=1')
+                    ->groupBy('e.id');
```

Comment:
The query on line 246 required having the following lines added in order to function:

```
                  ->leftJoin('e', sprintf('(%s)', $qbcut->getSQL()), 'cut', 'e.id = cut.channel_id')
                    ->leftJoin('e', MAUTIC_TABLE_PREFIX.'lead_donotcontact', 'dnc', 'e.id = dnc.channel_id and dnc.channel=\'email\' and dnc.reason=1')
                    ->groupBy('e.id, es.date_sent, es.date_read, es.email_address');
```